### ISSUE TITLE
Feature: Clean up tests

### ISSUE THREAD

**Issue by cindyli** [inclusive-design/adaptive-palette#187]
### Is your feature request related to a problem?

1. Tests grow with bug fixes and feature implementation. Some of them can be consolidated. 
2. Some cells that have disabled states have been changed from using `disabled` to `aria-disabled`. This change doesn't have test coverage.

### Describe the solution you'd like

1. Share common test functions, sync up test explanations
2. Add tests for palette cells using `aria-disabled`.
3. Define naming convention for test descriptions.

**Comment by cindyli** [comment]
Resolved by https://github.com/inclusive-design/adaptive-palette/pull/189
